gripper: {
  gripper_type: 'model_T42'
}
reflex: {
  proximal_finger_spring_coefficient: 5,
  distal_finger_spring_coefficient: 10,
  tendon_max_force: 300,
  finger_tendon_force_distribution: [1,0.5],
  lift_values: [0.0,0.2],
}
model_O: {
  proximal_finger_spring_coefficient: 5,
  distal_finger_spring_coefficient: 10,
  tendon_max_force: 300,
  finger_tendon_force_distribution: [1,0.5],
  lift_values: [-0.01,-0.21],
}
model_T42: {
  proximal_finger_spring_coefficient: 5,
  distal_finger_spring_coefficient: 10,
  tendon_max_force: 300,
  finger_tendon_force_distribution: [1,0.5],
  lift_values: [0,0.2],
}